### DISCUSSION TITLE
Is there latency added when running this script?

### DISCUSSION THREAD

**Discussion by Coarsen** [Falcosc/enable-loudness-equalisation#34] [General]
Curious, will this function for example just as my turtle beach headset that had the enhancements tab when connected to my Pc? Also will the compression function on my dac amp the same way it does on my turtle beach? Is this script just flagging the pc to add the same Loudness Eq?

**Comment by Falcosc** [comment]
Depends on your driver
It just adds the closed source post processing class id and enables the flag. https://github.com/Falcosc/enable-loudness-equalisation?tab=readme-ov-file#what-does-it-do

**Reply by Coarsen** [reply]
Alright cool, thank you.
